tests/e2e/specs/setup-mc/step-1-accounts.test.js

```diff
@@ -357,6 +357,96 @@ test.describe( 'Set up accounts', () => {
 				).toBeVisible();
 			} );
 		} );
+
+		test.describe( 'Claim Google Ads Account adsaccountclaim', () => {
+			test.beforeEach( async () => {
+				await setUpAccountsPage.mockJetpackConnected();
+				await setUpAccountsPage.mockGoogleConnected();
+				await setUpAccountsPage.mockMCConnected();
+
+				await setUpAccountsPage.fulfillAdsConnection( {
+					id: 12345,
+					currency: 'USD',
+					status: 'incomplete',
+					step: 'account_access',
+					sub_account: true,
+					symbol: '$',
+				} );
+
+				await setUpAccountsPage.fulfillAdsAccounts( {
+					id: 12345,
+					name: 'gla',
+				} );
+
+				await setUpAccountsPage.fulfillAdsAccountStatus( {
+					has_access: false,
+					invite_link: 'https://example.com',
+					step: 'account_access',
+				} );
```

Comment:
We have a mock for this.

```suggestion
				await setUpAccountsPage.mockAdsStatusNotClaimed();
```